src/pymanopt/manifolds/positive_definite.py

```diff
@@ -113,20 +91,221 @@ def exp(self, point, tangent_vector):
 
     def retraction(self, point, tangent_vector):
         p_inv_tv = np.linalg.solve(point, tangent_vector)
-        return multisym(point + tangent_vector + tangent_vector @ p_inv_tv / 2)
+        return multiherm(
+            point + tangent_vector + tangent_vector @ p_inv_tv / 2
+        )
 
     def log(self, point_a, point_b):
         c = np.linalg.cholesky(point_a)
         c_inv = np.linalg.inv(c)
         logm = multilogm(
-            c_inv @ point_b @ multitransp(c_inv),
+            c_inv @ point_b @ multihconj(c_inv),
             positive_definite=True,
         )
-        return c @ logm @ multitransp(c)
+        return c @ logm @ multihconj(c)
 
     def zero_vector(self, point):
         k = self._k
         n = self._n
         if k == 1:
             return np.zeros((n, n))
         return np.zeros((k, n, n))
+
+
+class SymmetricPositiveDefinite(_positive_definite):
+    """Manifold of symmetric positive definite matrices.
+
+    Points on the manifold and tangent vectors are represented as arrays of
+    shape ``k x n x n`` if ``k > 1``, and ``n x n`` if ``k == 1``.
+
+    Args:
+        n: The size of matrices in the manifold, i.e., the number of rows and
+            columns of each element.
+        k: The number of elements in the product geometry.
+
+    Note:
+        The geometry is based on the discussion in chapter 6 of [Bha2007]_.
+        Also see [SH2015]_ for more details.
+
+        The second-order retraction is taken from [JVV2012]_.
+    """
+
+    def __init__(self, n: int, *, k: int = 1):
+        self._n = n
+        self._k = k
+
+        if k == 1:
+            name = f"Manifold of positive definite {n}x{n} matrices"
+        else:
+            name = (
+                f"Product manifold of {k} positive definite {n}x{n} matrices"
+            )
+        dimension = int(k * n * (n + 1) / 2)
+        super().__init__(name, dimension)
+
+
+class HermitianPositiveDefinite(_positive_definite):
+    """Manifold of hermitian positive definite matrices.
+
+    Points on the manifold and tangent vectors are represented as arrays of
+    shape ``k x n x n`` if ``k > 1``, and ``n x n`` if ``k == 1``.
+
+    Args:
+        n: The size of matrices in the manifold, i.e., the number of rows and
+            columns of each element.
+        k: The number of elements in the product geometry.
+    """
+
+    def __init__(self, n: int, *, k: int = 1):
+        self._n = n
+        self._k = k
+
+        if k == 1:
+            name = (
+                f"Manifold of Hermitian positive definite ({n} x {n}) matrices"
+            )
+        else:
+            name = f"Product manifold of {k} ({n} x {n}) Hermitian positive definite"
+        dimension = int(k * n * (n + 1))
+        super().__init__(name, dimension)
+
+    def random_point(self):
```

Comment:
Shouldn't it still be possible to use this in `_PositiveDefiniteBase` and then leave it unchanged in `HermitianPositiveDefinite`, and replace the definition in `SymmetricPositiveDefinite` with this?
```python
def random_point(self):
    return super.random_point().real
```

Reply:
The real part of an HPD matrix $P$ is $(P+\overbar{P})/2$ so it should work!